### DISCUSSION TITLE
Gitter rooms

### DISCUSSION THREAD

**Discussion by Navid200** [Navid200/xDrip#163] [General]
@jamorham
@tolot27
@tzachi-dar

I cannot sign into Gitter.

Are others having the same problem?
Or, is it only my setup?

**Comment by tzachi-dar** [comment]
It says internal server error

בתאריך יום ה׳, 23 בדצמ׳ 2021, 00:05, מאת Navid ***@***.***>:

**Reply by Navid200** [reply]
Good to know it's not only me.

Thanks

**Comment by tolot27** [comment]
I have the same problem. As Gitter is [acquired by Element](https://blog.gitter.im/2020/09/30/gitter-element-acquisition), we can think of moving to Matrix, at least with the dev channel(s). The normal channel could become obsolete as better this discussions get accepted.

**Reply by Navid200** [reply]
I have already created a Discord room.
But, I need to find a way to reach all of you to pass the invitation.

Are you saying Gitter may not come back up?

**Reply by tolot27** [reply]
> I have already created a Discord room. But, I need to find a way to reach all of you to pass the invitation.

Yep, Discord is another platform we discussed already. Unfortunately, it is blogged by the proxy of my institute. But that's okay. I've to focus on science if I'm at work and not on xDrip. 😏 

> Are you saying Gitter may not come back up?

No, I did not say that. Currently, just the single-sign-on is broken.

**Reply by Navid200** [reply]
OK
Let's post here if anyone can sign into Gitter.
When that happens, we can discuss, in the Gitter room, an alternative channel in case this happens again.

Thanks
